include/swift/Demangling/DemangleNodes.def

```diff
@@ -119,6 +119,7 @@ NODE(MergedFunction)
 NODE(Metatype)
 NODE(MetatypeRepresentation)
 NODE(Metaclass)
+NODE(MethodLookupFunction)
```

Comment:
Don't forget to update ABI.rst too!

Reply:
Old-timer alert: it's actually ABI/Mangling.rst these days